streamalert/shared/config.py

```diff
@@ -218,9 +218,37 @@ def load_config(conf_dir='conf/', exclude=None, include=None, validate=True):
     if validate:
         _validate_config(config)
 
+    if config.get('logs'):
+        config['logs'] = _sanitize_logs_name(config['logs'])
+
+    if (config.get('global')
```

Comment:
eeek conditional blocks like this make me cringe so hard. readability is so bad. can be break this up into multiple statements?

like:

```python
infra_config = config.get('global', {}).get('infrastructure')
if not infra_config:
    return config

if 'firehose' not in infra_config:
    return config

# ... and so on ...
```

Reply:
Alternative way is to lump it into a function>

```
if self._requires_sanitized_log_names(etc):
  return config['global']....

return config
```